ghost/portal/src/components/common/ProductsSection.js

```diff
@@ -18,7 +18,7 @@ export const ProductsSectionStyles = ({site}) => {
         .gh-portal-products-pricetoggle {
             position: relative;
             display: flex;
-            background: #F3F3F3;
+            background: var(--grey13);
```

Comment:
There's a subtle but very important difference between the `#F3F3F3` and `var(--grey13)` which makes the contrast worse when using `var(--grey13)`. Let's keep the hard coded value.